packages/screenreaderonly/src/react/__stories__/index.story.js

```diff
@@ -0,0 +1,12 @@
+import { storiesOf } from '@storybook/react'
+import React from 'react'
+
+import ScreenReaderOnly from '../index.js'
+
+storiesOf('ScreenReaderOnly', module).add('basic', _ => (
+  <div>
+    <p>you should not see the following</p>
+
+    <ScreenReaderOnly />
```

Comment:
I do see the text.  The story usage is surprising.  I would have thought this component would wrap children.

<img width="292" alt="Screen Shot 2019-12-05 at 3 37 18 PM" src="https://user-images.githubusercontent.com/552044/70280241-3d94dd00-1775-11ea-97d2-d8acfbe8ce10.png">

Reply:
oops. i didn't do that right